CPS-0001/README.md

```diff
@@ -0,0 +1,49 @@
+    - https://github.com/cardano-foundation/CIPs/tree/master/CIP-0026
+    - https://github.com/cardano-foundation/CIPs/pull/355
+    - https://github.com/cardano-foundation/CIPs/pull/85
+    - https://github.com/cardano-foundation/CIPs/pull/112
+    - https://github.com/cardano-foundation/CIPs/pull/137
+    - https://github.com/cardano-foundation/CIPs/pull/299
+Created: 2022-19-10
+---
+
+## **Abstract**
+This document attempts to describe the problem in the Cardano ecosystem where there are many different types **subjects** or *chain entities* and there's a need to associate metadata with them. This metadata can be used to describe scripts, stake pools, script hashes, token policies and applications. This metadata can be used to provide information to the user on it's applications, it's trustworthiness and how to interact with it.
+
+## **Problem**
+**`Discoverability`**: Means to discover the different metadata claims associated with a subject. Discoverability is important for wallets, applications (i.e dapps, stores, etc) and users to be able to find the different metadata claims associated with a subject. This is important for the user to be able to make an informed decision on how to interact with a subject.
+
+**`Correctness`**: Lack of mechanism to assert that a given metadata claim is correct. Anybody and any service can provide metadata structures, but it's necessary to attest for the correctness of this metadata so that the user (or applications) are be able to make an informed decision in accepting, rejecting or how to interact with a subject.
+
+**`Trust`**: There's no standard mechanism to assert that the metadata is coming from a legitimate source. This is important for the user to be able to make an informed decision on how to interact with a subject. This is also important for applications to be able to provide a better user experience. 
+
+## **Use Cases**
+- Find scripts used by a dApp
+- Discover general information of a dApp (i.e name, description, icons, etc)
+- Find the different metadata claims associated with a subject
+- Associate some form of identity to a claim's owner so that it can be verified cryptographically
+- Offer mechanism to attest for the correctness of a given metadata object that can be fetched by wallets and applications from off-chain sources (i.e CIP-26 complaint servers)
+- Discover datum schemas used by a script in a specific context or dApp use cases
+- A wallet receiving a request to connect to a dApp, it can verify the authenticity of the dApp and the metadata associated with it
+
+## **Goals**
+- Define how metadata can be associated with a subject (i.e. dApp, stake pool, token policy, etc.)
+- Metadata should be discoverable by wallets and applications
+- Wallets and applications should be able to verify the correctness of metadata claims
+- Associate some form of identity to a metadata claim
+
+## **Open Questions**
+- Is this a combination of problems and should be split into multiple CPS?
```

Comment:
> Is this a combination of problems and should be split into multiple CPS?

It might make sense to split this problem by metadata subject i.e. stake pools, tokens, scripts, etc. As the problem can be solved quite differently for each of these entities. 

Without the historical context for CPSs its hard to know how wide the scope _should_ be.